lib/features/login/presentation/pages/home.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import '../../presentation/widgets/photo_holder.dart';
+import '../../presentation/widgets/home_page_buttons.dart';
```

Comment:
I know we don't have a style guide in place yet, but some rules I find very useful right off the bat like `trailing commas` and `separating external packages from internal files imports`. What do you think? If you agree, the lines above would go like:

```suggestion
import 'package:flutter/material.dart';
import 'package:flutter/widgets.dart';

import '../../presentation/widgets/photo_holder.dart';
import '../../presentation/widgets/home_page_buttons.dart';
```

Reply:
This is a good idea. Also to figure out our style guide. I have never really used on in flutter.

Reply:
Agreeed on this for readability